boot/freeldr/bootsect/ntfs.S

```diff
@@ -0,0 +1,829 @@
+/*
+ * PROJECT:         ReactOS Bootsector
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * FILE:            boot/freeldr/bootsect/fat32.S
+ * PURPOSE:         Implementing boot sector for NTFS
+ * COPYRIGHT:       Copyright 2020 Sylvain Deverre (deverre.sylv@gmail.com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <asm.inc>
+#include <freeldr/include/arch/pc/x86common.h>
+
+#define BP_REL(x) [bp+x-offset start]
+
+.code16
+
+//ORG HEX(7c00)
+
```

Comment:
(Regarding a previous comment by @Extravert-ir ): Yes the publicly available documentation on the format of the NTFS boot sector shows that all these fields are indeed part of the NTFS BPB.